Cabal/src/Distribution/Compat/CreatePipe.hs

```diff
@@ -1,81 +1,5 @@
-{-# LANGUAGE CPP #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE RankNTypes #-}
-
 module Distribution.Compat.CreatePipe (createPipe) where
```

Comment:
Should this module get a deprecation warning, maybe?

Reply:
Done.